src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/BaseEstimatorFromFootIMU.h

```diff
@@ -30,33 +30,52 @@ struct BaseEstimatorFromFootIMUState
 {
     manif::SE3d basePose; /**< final output of the estimator - pose of the robot
                              root link. */
+    manif::SE3d::Tangent baseVelocity; /**< velocity of the robot  root link. */
+    manif::SE3d footPose_L; /**< pose of the left foot */
+    manif::SE3d footPose_R; /**< pose of the right foot */
+    Eigen::Vector3d centerOfMassPosition;
```

Comment:
I would avoid propagating the CoM position. In the end the base estimator should provide the base state